### PR TITLE
Require specific SCAPtimony version.

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: foreman_openscap.gemspec [hunk 1]
```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
   s.test_files = Dir["test/**/*"]
 
   s.add_dependency 'deface', '~> 0'
-  s.add_dependency 'scaptimony', '~> 0'
+  s.add_dependency 'scaptimony', ['>= 0.3.0', '< 0.4.0']
```
Comment (shlomizadok) [inline foreman_openscap.gemspec]: Won't ```~> 0.3``` do the job?

Comment (isimluk) [inline foreman_openscap.gemspec]: In fact, the `~> 0.3.0` is needed. The `~> 0.3` has different semantics. That only shows it, how this notation is confusing.

Comment (ares) [inline foreman_openscap.gemspec]: definitely for non-rubyist, IMHO rubyists will find `['>= 0.3.0', '< 0.4.0']` weird and will start to think why you used this... anyway it's a small detail

**Comment by shlomizadok** [comment]
excellent. merging in 3, 2, 1....

**Pull request merged**